packages/@aws-cdk/aws-rds/lib/database-secret.ts

```diff
@@ -14,6 +14,16 @@ export interface DatabaseSecretProps {
    */
   readonly username: string;
 
+  /**
+   * The name of the secret.
+   *
+   * For "owned" secrets, this will be the full resource name (secret name + suffix), unless the
+   * '@aws-cdk/aws-secetsmanager:parseOwnedSecretName' feature flag is set.
```

Comment:
I'm sorry. I did not consider this point very deeply, and used the annotations on the Secret itself.

https://github.com/hedrall/aws-cdk/blob/03fba9619d7f3ad4f5a27d8123ff826a0ed26bce/packages/%40aws-cdk/aws-secretsmanager/lib/secret.ts#L33-L39

I would like to look into this more, including how it relates to the Feature flag.

Reply:
It's better to steal the documentation from `SecretProps`, rather than from `ISecret`.

https://github.com/hedrall/aws-cdk/blob/03fba9619d7f3ad4f5a27d8123ff826a0ed26bce/packages/%40aws-cdk/aws-secretsmanager/lib/secret.ts#L123-L129

Reply:
Thank you.

> A name for the secret. Note that deleting secrets from SecretsManager does not happen immediately, but after a 7 to * 30 days blackout period. During that period, it is not possible to create another secret that shares the same name.

However, I just noticed that this seems to be an old specification.
I tried destroying the deployed secret and renaming the deployed secret, but in both cases the secret was deleted immediately.
I think that `cloud formation` use a flag like as cli 'force-delete-without-recovery’.

So, I change comment simply.
( Should I also change the comment of SecretProps? )